tests/PactContinuationSpec.hs

```diff
@@ -338,16 +330,12 @@ testNoRollbackFunc mgr = do
   allResults       <- runAll mgr [moduleCmd, executePactCmd, contNextStepCmd,
                               noRollbackCmd, checkStateCmd]
 
-  let moduleCheck        = makeCheck moduleCmd False Nothing
-      executePactCheck   = makeCheck executePactCmd False $ Just "step 0"
-      contNextStepCheck  = makeCheck contNextStepCmd False $ Just "step 1"
-      noRollbackCheck    = makeCheck noRollbackCmd False $ Just "No rollback on step 1" -- not a failure
-      checkStateCheck    = makeCheck checkStateCmd True $
-                           pactIdNotFoundMsg executePactCmd
-      allChecks          = [moduleCheck, executePactCheck, contNextStepCheck,
-                            noRollbackCheck, checkStateCheck]
-
-  allResults `shouldMatch` allChecks
+  runResults allResults $ do
+    moduleCmd `succeedsWith`  Nothing
+    executePactCmd `succeedsWith` Just "step 0"
+    contNextStepCmd `succeedsWith` Just "step 1"
+    noRollbackCmd `failsWith` Just "(step \"step 1\")   : Failure: Rollback requested but none in step"
```

Comment:
@LindaOrtega the old test would try to rollback which would _succeed_ with "no rollback" message (weird) and then fail on step 2 afterward. It seems like Pact displays the correct behavior now, which is to simply reject the improper rollback attempt, and proceed to honor a valid step 2 request. Please confirm/comment.

Reply:
So the rational for this test was: 
```
context "when trying to rollback a step without a rollback function" $
    it "outputs that no rollback function exists for step and deletes pact from state" 
```

I looked through the emails from when I was coding this up, and found that this was actually an unexpected behavior:

```
1. No rollback function: We've discussed this one and I've added a comment on the rollback continuation code. The issue is that when a step is rolled back and it doesn't have a rollback function, no error is thrown. applyContinuation therefore treats it as a success and delete that pact from the state. 
```
I think [`evalContinuation`](https://github.com/kadena-io/pact/blob/master/src-ghc/Pact/Server/PactService.hs#L145) didn't throw an exception at that time for this scenario. See this comment in [PactService](https://github.com/kadena-io/pact/blob/master/src-ghc/Pact/Server/PactService.hs#L160).

So rolling back a step without a rollback function seemed liked unexpected behavior. So your fix seems to be the correct behavior.

Reply:
The description here
https://github.com/kadena-io/pact/pull/496/files#diff-dd34ef6e2d4d59bae40983e261d98c43R234
should then change to reflect this new behavior

Reply:
That link doesn't work, maybe put a dedicated comment where you want the change @LindaOrtega

Reply:
It doesn't let me comment since you didn't change that section. But its line 234.

Replace this:
```
context "when trying to rollback a step without a rollback function" $
    it "outputs that no rollback function exists for step and deletes pact from state" $
```
with this:
```
context "when trying to rollback a step without a rollback function" $
    it "outputs a rollback failure and doesn't change the pact's state" $
```

Reply:
Thanks, fixed